hishel/_async/_storages.py

```diff
@@ -84,6 +88,8 @@ def __init__(
 
         self._file_manager = AsyncFileManager(is_binary=self._serializer.is_binary)
         self._lock = AsyncLock()
+        self._check_ttl_every = check_ttl_every
+        self._timer = time.time()
```

Comment:
```suggestion
        self._timer = time.monolitic()
```

Let's use [monolitic](https://docs.python.org/3/library/time.html#time.monotonic) here

Reply:
And also we can rename it to `last_clean` which maybe makes more sense.